openedx/features/redhouse_panel/api/v0/tests/test_views.py

```diff
@@ -0,0 +1,87 @@
+from django.conf import settings
+from django.test import TestCase
+from django.test.client import RequestFactory
+from django.urls import reverse
+from rest_framework import status
+
+from openedx.core.djangoapps.site_configuration.tests.factories import SiteConfigurationFactory, SiteFactory
+from openedx.features.edly import cookies
+from openedx.features.edly.tests.factories import EdlySubOrganizationFactory, EdlyUserFactory
+from student.tests.factories import UserFactory
+
+
+class SiteViewTests(TestCase):
+    USERNAME = "edx"
+    EMAIL = "edx@example.com"
+    PASSWORD = "edx"
+
+    def setUp(self):
+        self.staff = UserFactory.create(
+            username=self.USERNAME,
+            email=self.EMAIL,
+            password=self.PASSWORD,
+            is_staff=True
+        )
+        self.client.login(username=self.USERNAME, password=self.PASSWORD)
+        self.site_name = 'fake.name'
+        self.site_address = 'fake.address'
+
+    def test_return_site_successfully(self):
+        site = SiteFactory(name=self.site_name)
+        site_configuration = SiteConfigurationFactory(values={'address': self.site_address}, site=site)
+
+        url = reverse('redhouse_features:redhouse_panel:redhouse_panel_api.v0:site', kwargs={'pk': site.id})
```

Comment:
Due to the recent change in the base, I think you might have to update the reverse url

Reply:
@danialmalik fixed it